galaxy/accounts/tests/__init__.py

```diff
@@ -1,5 +1,3 @@
-# (c) 2012-2018, Ansible by Red Hat
```

Comment:
What is the reason for copyright removal?

Reply:
In this case git decided that I moved file, but it's new actually.